project/cmake/CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ endif()
 
 # Optional dependencies
 set(optional_deps MicroHttpd MySqlClient SSH XSLT
-                  Alsa UDEV DBus Avahi
+                  Alsa UDEV DBus Avahi SmbClient
```

Comment:
enable switch for optional deps was changed in https://github.com/xbmc/xbmc/commit/fe50635fb7a09d1576890a5d548429d55af00221